nextjs-task-system/app/groups/page.tsx

```
@@ -0,0 +1,127 @@
+  };
+
+  return (
+    <ProtectedRoute>
+      <div className="grid grid-cols-1 gap-1 p-4 sm:grid-cols-2 lg:grid-cols-3">
+        {groups.map((group) => (
+          <CustomCardGroup
+            user={user}
+            token={token}
+            group={group}
+            openDeleteModal={() => setTaskToDelete(group.id!)}
+          />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)